scripts/manage_dependencies.py

```diff
@@ -0,0 +1,114 @@
+import argparse
```

Comment:
can we type this file?

Reply:
I'm not sure what you mean, it's a module in the standard API of Python. Do you want us to add a stub file ?

Reply:
I can't comment on the entire file, I just mean can we type hint all the functions in the `manage_dependencies.py` file 🙂

Reply:
Done. Sorry for the misunderstanding!